ui/dashboards/src/components/AddPanel/AddPanel.tsx

```diff
@@ -41,26 +44,42 @@ const AddPanel = ({ isOpen, onClose }: AddPanelProps) => {
   const [group, setGroup] = useState(0);
   const [panelName, setPanelName] = useState('');
   const [panelDescription, setPanelDescription] = useState('');
+  const [kind, setKind] = useState('');
+  const [options, setOptions] = useState<JsonObject>({});
 
-  const onSelectGroupChange = (e: SelectChangeEvent<number>) => {
-    setGroup(e.target.value as number);
+  const handleGroupChange: SelectProps<number>['onChange'] = (e) => {
+    const { value } = e.target;
+
+    // Handle string (which would be empty string but shouldn't happen since we don't allow a "None" option) by
+    // just ignoring it
+    if (typeof value === 'string') return;
+
+    setGroup(value);
   };
 
-  const onPanelNameChange = (e: ChangeEvent<HTMLInputElement>) => {
+  const handlePanelNameChange = (e: ChangeEvent<HTMLInputElement>) => {
     setPanelName(e.target.value);
   };
 
-  const onPanelDescriptionChange = (e: ChangeEvent<HTMLInputElement>) => {
+  const handlePanelDescriptionChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Should this populate the description text in the InfoTooltip (I'm not seeing the little 'i' icon after adding a new panel)?

Reply:
Ohh nvm, I see now--the info icon only shows when aren't in Edit Mode